src/libcollections/vec.rs

```diff
@@ -659,6 +659,43 @@ impl<T> Vec<T> {
         }
     }
 
+
+    /// Move all elements from one vector to another
+    ///
+    /// # Panics
+    ///
+    /// Panics if the number of elements in the vector overflows a `uint`.
+    ///
+    /// # Examples
+    /// ```rust
+    /// let mut vec = vec![1i, 2, 3];
+    /// let mut vec2 = vec![4i, 5, 6];
```

Comment:
Integer fallback should mean these integer suffixes aren't necessary. Should choose i32 automatically.